web/ui/www/index.html

```diff
@@ -9,9 +9,11 @@
         <title>Whisper</title>
 
         <!-- Bootstrap core CSS -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css" />
```

Comment:
@claudiosegala, we are not using cdn provided scripts (css or js). We always download them to the `/static/css` or `/static/js` folders.